From 0d672945dfca506d4e49e857ce886d2b3dc80e96 Mon Sep 17 00:00:00 2001
From: Nathan West <nathan.west@gnuradio.org>
Date: Mon, 1 Aug 2016 17:12:24 -0400
Subject: [PATCH 04/18] apps: fix profile update reading end of lines

---
 apps/volk_profile.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apps/volk_profile.cc b/apps/volk_profile.cc
index 2086e3f..51591cc 100644
--- a/apps/volk_profile.cc
+++ b/apps/volk_profile.cc
@@ -261,7 +261,7 @@ void read_results(std::vector<volk_test_results_t> *results, std::string path)
                     found = 127;
                 }
                 str_size = config_str.size();
-                char buffer[128];
+                char buffer[128] = {'\0'};
                 config_str.copy(buffer, found + 1, 0);
                 buffer[found] = '\0';
                 single_kernel_result.push_back(std::string(buffer));
-- 
2.11.0

