From 0f6d889b891bc2ac78c56ad18e43c4ec5a372574 Mon Sep 17 00:00:00 2001
From: Nathan West <nathan.west@gnuradio.org>
Date: Thu, 4 Aug 2016 11:30:55 -0400
Subject: [PATCH 05/18] qa: lower tolerance for 32fc_mag to fix issue #96

---
 lib/kernel_tests.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/kernel_tests.h b/lib/kernel_tests.h
index 2bf1f0c..7c82733 100644
--- a/lib/kernel_tests.h
+++ b/lib/kernel_tests.h
@@ -24,6 +24,8 @@ std::vector<volk_test_case_t> init_test_list(volk_test_params_t test_params)
     // Some kernels need a lower tolerance
     volk_test_params_t test_params_inacc = volk_test_params_t(1e-2, test_params.scalar(),
             test_params.vlen(), test_params.iter(), test_params.benchmark_mode(), test_params.kernel_regex());
+    volk_test_params_t test_params_inacc_tenth = volk_test_params_t(1e-1, test_params.scalar(),
+            test_params.vlen(), test_params.iter(), test_params.benchmark_mode(), test_params.kernel_regex());
     volk_test_params_t test_params_int1 = volk_test_params_t(1, test_params.scalar(),
             test_params.vlen(), test_params.iter(), test_params.benchmark_mode(), test_params.kernel_regex());
 
@@ -79,7 +81,7 @@ std::vector<volk_test_case_t> init_test_list(volk_test_params_t test_params)
         (VOLK_INIT_TEST(volk_32fc_index_max_16u,      volk_test_params_t(3, test_params.scalar(), test_params.vlen(), test_params.iter(), test_params.benchmark_mode(), test_params.kernel_regex())))
         (VOLK_INIT_TEST(volk_32fc_index_max_32u,      volk_test_params_t(3, test_params.scalar(), test_params.vlen(), test_params.iter(), test_params.benchmark_mode(), test_params.kernel_regex())))
         (VOLK_INIT_TEST(volk_32fc_s32f_magnitude_16i,                   test_params_int1))
-        (VOLK_INIT_TEST(volk_32fc_magnitude_32f,                        test_params_inacc))
+        (VOLK_INIT_TEST(volk_32fc_magnitude_32f,                        test_params_inacc_tenth))
         (VOLK_INIT_TEST(volk_32fc_magnitude_squared_32f,                test_params))
         (VOLK_INIT_TEST(volk_32fc_x2_multiply_32fc,                     test_params))
         (VOLK_INIT_TEST(volk_32fc_x2_multiply_conjugate_32fc,           test_params))
-- 
2.11.0

