From 18ed0f15bce6053fee5405fbe7077760a95788a3 Mon Sep 17 00:00:00 2001
From: Pascal Giard <pascal.giard@lacime.etsmtl.ca>
Date: Fri, 28 Aug 2015 13:12:37 -0400
Subject: [PATCH 06/11] Fix ifdef guard to use the same naming convention as
 others.

---
 include/volk/volk_complex.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/volk/volk_complex.h b/include/volk/volk_complex.h
index 5bd9250..83812ea 100644
--- a/include/volk/volk_complex.h
+++ b/include/volk/volk_complex.h
@@ -1,5 +1,5 @@
-#ifndef INCLUDE_VOLK_COMPLEX_H
-#define INCLUDE_VOLK_COMPLEX_H
+#ifndef INCLUDED_VOLK_COMPLEX_H
+#define INCLUDED_VOLK_COMPLEX_H
 
 /*!
  * \brief Provide typedefs and operators for all complex types in C and C++.
-- 
2.1.4

