From 46a2bd514a1392ca2404d2123e5378a50357b79b Mon Sep 17 00:00:00 2001
From: Pascal Giard <pascal.giard@lacime.etsmtl.ca>
Date: Fri, 28 Aug 2015 13:27:18 -0400
Subject: [PATCH 10/11] Adds a bit256 union (especially useful for AVX/AVX2).

---
 include/volk/volk_common.h | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/include/volk/volk_common.h b/include/volk/volk_common.h
index 7180b6c..02c5bd9 100644
--- a/include/volk/volk_common.h
+++ b/include/volk/volk_common.h
@@ -88,6 +88,21 @@ union bit128{
   #endif
 };
 
+union bit256{
+  uint8_t i8[32];
+  uint16_t i16[16];
+  uint32_t i[8];
+  float f[8];
+  double d[4];
+
+  #ifdef LV_HAVE_AVX
+  __m256 float_vec;
+  __m256i int_vec;
+  __m256d double_vec;
+  #endif
+};
+
 #define bit128_p(x) ((union bit128 *)(x))
+#define bit256_p(x) ((union bit256 *)(x))
 
 #endif /*INCLUDED_LIBVOLK_COMMON_H*/
-- 
2.1.4

